addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -156,7 +156,7 @@ def _compute_l10n_latam_available_document_types(self):
     @api.depends_context('internal_type')
     def _compute_l10n_latam_document_type(self):
         internal_type = self._context.get('internal_type', False)
-        for rec in self.filtered(lambda x: x.state == 'draft' and x.l10n_latam_available_document_type_ids):
+        for rec in self.filtered(lambda x: x.state == 'draft'):
```

Comment:
Why?

Reply:
Hi @jco-odoo 
Here is the reason why of this change https://drive.google.com/file/d/1H7g78ANNoxG28p4ioX77FqEIbUOAcMm2/view

Reply:
I don't like the fact that the domain is sometimes empty, but anyways, this makes sense.